### ISSUE TITLE
Keyboard shows when two way binding property in ViewModel change

### ISSUE THREAD

**Issue by pazunino** [UweReisewitz/XamarinAndroidEntry#3]
Hi @UweReisewitz ,
I have a case that is needed to change the content of an ExtendedEntry from ViewModel via the binding property.
Although the ShowVirtualKeyboardOnFocus property is set to false, the keyboard shows and stays up.
Regards
Pedro

**Comment by UweReisewitz** [comment]
Which Android version are you on?

**Comment by pazunino** [comment]
I have 2 devices, one with 7.1.2 and the other with 10. In both devices the behavior is the same.
As a workaround I set the focus on the extendedentry after changing the VM property with a trigger.
In the 7.1.2 the workaround works fine the keyboard is not noticeable, but in the device with 10 the keyboard goes up and down in a very noticeable way.

**Comment by UweReisewitz** [comment]
Do you have a repo for me? That would help a lot.

**Comment by pazunino** [comment]
No, I don't have. I'll try to find a slot of time and make one cutting down my app. 
The VM property takes the value from a rfid tag reading and shows for validation in the View. I have to take out this nfc library.

**Comment by UweReisewitz** [comment]
I'm sorry for the delay. I will have to investigate this (and I need it for my own project, too).
But due to other priorities that will take a few weeks.

**Comment by pazunino** [comment]
Sorry for the delayed response. That's ok, no problem I can wait, I am using the workaround explained above. The flashing keyboard is not a major problem for the user. Thank you for your work.
